src/intervals/hyperbolic_functions.jl

```diff
@@ -56,6 +56,6 @@ for f in (:tanh, :asinh, :acosh, :atanh)
     @eval function ($f)(a::Interval{Float64})
         isempty(a) && return a
 
-        float(($f)(big53(a)))
+        float( $f (big53(a)) )
```

Comment:
I get the following warning due to this line:
```
WARNING: deprecated syntax "$f (" at /Users/benet/Fisica/6-IntervalArithmetics/ValidatedNumerics.jl/src/intervals/hyperbolic_functions.jl:59.
Use "$f(" instead.
```
I guess the blank space is bothering...